akka-docs/rst/java/http/routing-dsl/directives/basic-directives/extractExecutionContext.rst

```diff
@@ -3,17 +3,10 @@
 extractExecutionContext
 =======================
 
-Signature
----------
-TODO: Add example snippet.
-.. 
-.. includecode2:: /../../akka-http/src/main/scala/akka/http/scaladsl/server/directives/BasicDirectives.scala
-   :snippet: extractExecutionContext
-
 Description
 -----------
 
-Extracts the ``ExecutionContext`` from the ``RequestContext``.
+Extracts the ``ExecutionContextExecutor`` from the ``RequestContext``.
```

Comment:
Hm... not sure, execution context is the akka actor system exposed type - so let's keep that, maybe adding one extract for the executor?